modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js

```diff
@@ -55,7 +55,11 @@ const Radio = ({
 	}, [predefinedValue]);
 
 	const [currentValue, setCurrentValue] = useSyncValue(
-		initialValue ? initialValue : predefinedValueMemo,
+		initialValue !== undefined &&
+			initialValue !== null &&
+			initialValue !== ''
+			? initialValue.toString()
+			: predefinedValueMemo,
```

Comment:
`undefined`, `null` and `''` are all falsy values. Are you sure that the following does not produce the same result?
`initialValue ? initialValue.toString() : predefinedValueMemo`?

Reply:
Hi @cgmonte,

I am afraid that your alternative will fail for the integer 0. This will return the predefinedValueMemo instead of the String '0'.

Reply:
You are right, thanks.